test/recipes/25-test_x509.t

```diff
@@ -10,13 +10,15 @@
 use strict;
 use warnings;
 
+use POSIX;
 use File::Spec;
 use OpenSSL::Test::Utils;
 use OpenSSL::Test qw/:DEFAULT srctop_file/;
+use Time::Piece;
```

Comment:
Likely this will not be needed here after moving the subroutines.

Reply:
It is still needed, but it was sufficient to move it to `tconversion.pl`.